hypertrace-core-graphql-span-schema/src/main/java/org/hypertrace/core/graphql/span/request/DefaultSpanRequestBuilder.java

```diff
@@ -54,12 +68,13 @@ public Single<SpanRequest> build(
       Map<String, Object> arguments,
       List<AttributeExpression> spanAttributeExpressions,
       List<AttributeExpression> logAttributeExpressions) {
+
     return zip(
         resultSetRequestBuilder.build(
             context, HypertraceCoreAttributeScopeString.SPAN, arguments, spanAttributeExpressions),
         logEventAttributeRequestBuilder.buildAttributeRequest(context, logAttributeExpressions),
         (resultSetRequest, logEventAttributeRequest) ->
-            new DefaultSpanRequest(context, resultSetRequest, logEventAttributeRequest));
+            new DefaultSpanRequest(context, resultSetRequest, logEventAttributeRequest, false));
```

Comment:
This is used via ExportSpans API, and `ExportSpanResult` doesn't extends `ResultSet`. It doesn't support total.
Ref: https://github.com/hypertrace/hypertrace-core-graphql/blob/main/hypertrace-core-graphql-span-schema/src/main/java/org/hypertrace/core/graphql/span/schema/ExportSpanResult.java#L15C1-L15C1

Reply:
> This is used via ExportSpans API, and ExportSpanResult doesn't extends ResultSet. It doesn't support total.

We can add it as a comment to explain why fetchTotal is set to false here.